smart_contracts/db_resource.cpp

```diff
@@ -0,0 +1,16 @@
+#include "sophiatx/smart_contracts/db_resource.h"
```

Comment:
Can we maintain the convention of naming c++ headers with "hpp" suffix?

Reply:
yes, my bad :)